components/experimental/src/duration/format.rs

```diff
@@ -677,20 +649,20 @@ mod tests {
     fn test_duration_formatter() {
         let duration = Duration {
             sign: DurationSign::Negative,
-            years: 1,
+            years: 0,
             months: 2,
             weeks: 3,
-            days: 0,
             hours: 12,
             minutes: 1,
             seconds: 5,
             milliseconds: 130,
             microseconds: 140,
-            nanoseconds: 0,
+            ..Default::default()
```

Comment:
good catch

Reply:
if the year is zero, yo do not need to have it too

Reply:
I left `year: 0` to emphasize the leading zero, since that is an important part of the test. Would you recommend removing it?

Reply:
in that case, just add a comment

```
   year : 0, // The leading part is equal to zero
```

or something like that, so no one in the future delete it.